diff --git a/Makefile b/Makefile
index b06bc25..3369704 100644
--- a/Makefile
+++ b/Makefile
@@ -62,7 +62,7 @@ all:
 	@echo "Invalid `TARGET variable ; valid values are: pluto, sidekiqz2" &&
 	exit 1
 else
-all: clean-build $(TARGETS) zip-all legal-info
+all: clean-build $(TARGETS) zip-all
 endif
 
 .NOTPARALLEL: all
@@ -118,10 +118,11 @@ buildroot/output/images/rootfs.cpio.gz:
 	@echo device-fw $(VERSION)> $(CURDIR)/buildroot/board/$(TARGET)/VERSIONS
 	@$(foreach dir,$(VSUBDIRS),echo $(dir) $(shell cd $(dir) && git describe --abbrev=4 --dirty --always --tags) >> $(CURDIR)/buildroot/board/$(TARGET)/VERSIONS;)
 	make -C buildroot ARCH=arm zynq_$(TARGET)_defconfig
+
 	make -C buildroot legal-info
 	scripts/legal_info_html.sh "$(COMPLETE_NAME)" "$(CURDIR)/buildroot/board/$(TARGET)/VERSIONS"
 	cp build/LICENSE.html buildroot/board/$(TARGET)/msd/LICENSE.html
-	make -C buildroot TOOLCHAIN_EXTERNAL_INSTALL_DIR=$(TOOLCHAIN_PATH) ARCH=arm CROSS_COMPILE=$(CROSS_COMPILE) BUSYBOX_CONFIG_FILE=$(CURDIR)/buildroot/board/$(TARGET)/busybox-1.25.0.config all
+	make -C buildroot TOOLCHAIN_EXTERNAL_INSTALL_DIR=$(TOOLCHAIN_PATH) ARCH=arm CROSS_COMPILE=$(CROSS_COMPILE) BUSYBOX_CONFIG_FILE=$(BR2_EXTERNAL)/board/$(TARGET)/busybox-1.25.0.config all
 
 .PHONY: buildroot/output/images/rootfs.cpio.gz
 
diff --git a/scripts/pluto.its b/scripts/pluto.its
index e69e417..aa7fa08 100644
--- a/scripts/pluto.its
+++ b/scripts/pluto.its
@@ -156,8 +156,8 @@
 		};
 
 		config@9 { /* This one is actually RevB */
-			description = "Linux with fpga RevB";
-			fdt = "fdt@2";
+			description = "Linux with fpga RevC";
+			fdt = "fdt@3";
 			kernel = "linux_kernel@1";
 			ramdisk = "ramdisk@1";
 			fpga = "fpga@1";
